process/block/shardblock.go

```diff
@@ -1588,3 +1595,59 @@ func (sp *shardProcessor) IsInterfaceNil() bool {
 	}
 	return false
 }
+
+func (sp *shardProcessor) addProcessedMiniBlock(metaBlockHash []byte, miniBlockHash []byte) {
```

Comment:
should be moved into a separate components - like MiniBlocksVerifier / Processor

Reply:
I would make this in other task. Added one task in Jira for this (EN-4168)